prometheus-exporters/aci-suite/charts/correlator/values.yaml

```diff
@@ -0,0 +1,78 @@
+fullname: aci-correlator
+enabled: true
+
+replicaCount: 1
+
+image:
+  name: aci-correlator
+  tag: "1.7.0"
+  pullPolicy: Always
+
+resources:
+  limits: {}
+  requests: {}
```

Comment:
explicitly name resources please and also add them in the chart accordingly. no `toYaml` in this case.

Applies to all.